Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8
+
+COPY . /app
+WORKDIR /app
+RUN apt update && apt install -yqq g++ gcc libc6-dev make pkg-config libffi-dev python3-dev git
+RUN pip3 install pipenv
+RUN pipenv install --system --deploy --ignore-pipfile
+RUN pip3 install -r requirements.txt
```

Comment:
There is no file named "requirements.txt"

Reply:
yeah I think this line needs to be removed. It will cause an error since we're not relying on pip anymore. Also, why ignore the pip file?